Classes/Service/ImageResizer.php

```diff
@@ -545,6 +560,13 @@ protected function expandValuesInRuleset(array $ruleset)
                         $value = '';
                     }
                     break;
+                case 'max_size':
+                    if (!is_numeric($value)) {
+                        $unit = strtoupper(substr($value, -1));
+                        $factor = 1 * ($unit === 'M' ? 1000000 : 1);
+                        $value = intval(trim(substr($value, 0, strlen($value) - 1))) * $factor;
```

Comment:
According to documentation you support `B`, `K` and `M` but you only implemented `B` and `M` here.

Reply:
No `B`, `K` and `M` suffix are only supported for file size threshold. The newly introduced image size limit only supports `M`.

see https://github.com/xperseguers/t3ext-image_autoresize/pull/11/files#diff-ed3b45d815392cd7dcc9cce6d18c4598R73

> The field accepts the suffice "M" for megapixels.

Reply:
OK, so this is about limiting the number of "mega pixels", that's what I thought as well, but when we speak about "image size", for me (and I assume for most users out there), size is a matter of "weight", not number of pixels, I guess the description is not average-user-friendly enough.

Reply:
Remember that for most users, documentation is something you don't read (anymore). So configuration options should be self-explanatory I'd say.

Reply:
Hmmm... yeah. It should already be consistent in the documentation (file size vs. image size).
Do you have an idea how to label the field? Would "Maximum image size (pixels)" be sufficient?

Reply:
Yes, something like that, even "Maximum image size (number of pixels)" just to be crystal-clear.